src/bika/lims/content/analysisspec.py

```diff
@@ -81,13 +81,19 @@
         widget=AnalysisSpecificationWidget(
             label=_("Specifications"),
             description=_(
-                "'Min' and 'Max' values indicate a valid results range. Any "
-                "result outside this results range will raise an alert. 'Min "
-                "warn' and 'Max warn' values indicate a shoulder range. Any "
-                "result outside the results range but within the shoulder "
-                "range will raise a less severe alert. If the result is out of "
-                "range, the value set for '< Min' or '< Max' will be displayed "
-                "in lists and results reports instead of the real result.")
+                u"description_analysisspec_resultsrange",
+                default=u"'Min' and 'Max' values indicate a valid results "
+                        u"range. Any result outside this results range will "
+                        u"raise an alert. 'Min warn' and 'Max warn' values "
+                        u"indicate a shoulder range. "
+                        u"Any result outside the results range but within the "
+                        u"shoulder range will raise a less severe alert. "
+                        u"If the result is out of range, the value set for "
```

Comment:
somehow the description in the UI is cropped at this point (Maybe because of the starting single quote in the next line?).

<img width="1396" src="https://github.com/senaite/senaite.core/assets/713193/8db6ff2c-b9e7-4b0c-bc96-9f04c881ff30">

Reply:
Cannot reproduce :thinking: . It is displayed correctly to me:

![Captura de 2023-09-01 14-39-34](https://github.com/senaite/senaite.core/assets/832627/60b794ba-7837-4cfa-b11f-49fb1021aaf6)

Reply:
Even when I shrink the window:

![Captura de 2023-09-01 14-41-27](https://github.com/senaite/senaite.core/assets/832627/d63af995-b6bd-4600-aaaf-787af8ab7ba2)

Reply:
Hmm, it does not seem like a browser issue, because the text is even cropped in the HTML:

<img width="543" alt="Add Analysis Specification — SENAITE LIMS 2023-09-01 2 PM-45-27" src="https://github.com/senaite/senaite.core/assets/713193/0bf25905-6503-4c42-bd81-ba9bca015f5c">

It appears to me even for new created sites when adding a new specification.

Reply:
Umm..

![Captura de 2023-09-01 15-10-28](https://github.com/senaite/senaite.core/assets/832627/d5563073-c470-430b-91dd-43dc52c5d51b)

These are my versions:

![Captura de 2023-09-01 15-11-27](https://github.com/senaite/senaite.core/assets/832627/173c8a77-08f7-40b1-bc5b-7f917cd4f679)